### DISCUSSION TITLE
Map de DraftBot

### DISCUSSION THREAD

**Discussion by RayZZo9** [Crownicles/Crownicles#1863] [Suggestions fermées]
Sur la map, on voit la route qu'on est en train de prendre. C'est bien. On peut faire mieux.

Ce que je propose c'est que l'icône qui représente le joueur se déplace avec le temps en fonction de notre place réelle dans le jeu. C'est à dire que Au début de notre voyage sur une route de 2h elle sera à un bout de la route, au bout d'une heure elle aura fait la moitié, après une demi-heure il y a une altération d'êtat, le marqueur stagne puis reprend son chemin.

Cela permet d'avoir une map active, un peu.

On pourrait même aller plus loin en mettant l'émoji caractérisant une l'altération d'êtat au dessus du marqueur si le joueur en a une.

Je ne me rend pas compte de la dificulté que cela demandera, donc, je suggère, et vous en fait ce que vous voulez.

**Comment by BastLast** [comment]
Hello, ça demanderai une semaine de développement à plein ou un serveur bcp plus puissant pr faire du traitement d'image. Surtout si on veut garder le fait que la map soit une image de haute qualité.

Je deplace dans refusé

**Comment by Pietagorh** [comment]
C'est assez complexe oui, mais c'est pas le plus important
J'parie que Drapht va dire que ça serait une cata niveau perfs
Là l'avantage c'est qu'on a juste à prégénérer les maps et après choisir la bonne, donc c'est mieux
